packages/core/upload/admin/src/pages/App/components/BulkMoveButton.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { useIntl } from 'react-intl';
+import { Button } from '@strapi/design-system/Button';
+import Folder from '@strapi/icons/Folder';
+import { getAPIInnerErrors } from '@strapi/helper-plugin';
+
+import { BulkMoveDialog } from '../../../components/BulkMoveDialog';
+import { AssetDefinition, FolderDefinition } from '../../../constants';
+import { useBulkMove } from '../../../hooks/useBulkMove';
+import { getTrad } from '../../../utils';
+
+export const BulkMoveButton = ({ selected, onSuccess }) => {
+  const { formatMessage } = useIntl();
+  const [showConfirmDialog, setShowConfirmDialog] = useState(false);
+  const [dialogErrors, setDialogErrors] = useState(null);
+  const { move } = useBulkMove();
+
+  const handleConfirmMove = async ({ moved, destinationFolderId } = {}) => {
+    try {
+      if (moved) {
+        await move(destinationFolderId, selected);
+        onSuccess();
+      }
+
+      setShowConfirmDialog(false);
+      // eslint-ignore-next-line no-empty
```

Comment:
I don't have a lint error when I remove this one, is it needed?

Reply:
Done in 6fe8603e0428b9c4c1a277b66963e52cb1a7e9de